griptape/drivers/vector/mongodb_vector_store_driver.py

```diff
@@ -81,10 +81,38 @@ def load_entries(
 
     def query(
         self,
-        query: str,
+        vector: list[float],
         count: Optional[int] = None,
         namespace: Optional[str] = None,
         include_vectors: bool = False,
+        skip: Optional[int] = 0,
         **kwargs
     ) -> list[BaseVectorStoreDriver.QueryResult]:
-        raise NotImplementedError("Vector search not supported in MongoDB")
+        collection = self.get_collection()
+
+        knn_k = skip + (count if count else 10)
```

Comment:
This took me some time to parse. I wonder if we should rename `skip` to `offset` in the method, so it's more in line with the meaning behind it. Shouldn't be an issue since we already use `count` to represent `k`.